### ISSUE TITLE
Bug: After pausing a container, view doesn't refresh

### ISSUE THREAD

**Issue by pcgeek86** [TomasTomecek/sen#143]
When I stop or start a container, the main view refreshes normally, as you would expect. However, when I pause or unpause a container, the view does not refresh as I would expect it to. Manually refreshing the view, using the `@` command works just fine. It'd be cool if this was fixed though!

**Comment by TomasTomecek** [comment]
Yeah, that sounds like a bug. What version of dockerd are you running? `$ docker version`

Sen listens to events from dockerd. If sen doesn't receive those regarding the pause/unpause actions, it won't update the main UI.

**Comment by TomasTomecek** [comment]
Okay, I can reproduce locally. I hope it's just an issue of sen and dockerd doesn't have anything to do with it.

I'll try to fix it & release in coming days or week.

**Comment by TomasTomecek** [comment]
I finally found time to look into this: the problem is that dockerd is not sending a real-time event when the container is in paused state. It sends the event when you ask it to pause.

I created a terrible workaround: sleep for a second when the events pause/unpause; no idea if that would work for machines with higher load. On my current laptop with 5 images and 2 containers it worked well. I don't use docker lately much.
